app/models/user.rb

```diff
@@ -364,8 +364,8 @@ def cached_followed_tag_names
   end
 
   # methods for Administrate field
```

Comment:
Unrelated, but since we're here already, should we maybe get rid of this comment, we no longer use Administrate.